test/e2e/utils/recordedit-helpers.js

```diff
@@ -390,6 +472,95 @@ exports.testPresentationAndBasicValidation = function(tableParams, isEditMode) {
                     });
                 });
             }
+            
+            if (jsonCols.length > 0) {
+            describe("JSON fields, ", function () {
```

Comment:
This test case is different from other column types. Try to make it consistent with other types. Take a look at other column types, for example the [markdown](https://github.com/informatics-isi-edu/chaise/blob/20189a1a052f85941fb30b6fd0a2962d62f0f986/test/e2e/utils/recordedit-helpers.js#L407,L474) that Amit wrote.